### PR TITLE
[UI refactor] Remove .bind() part 4 - Remove all .bind() in RAI repo and set rule to block adding .bind()

### PR DESCRIPTION
**Pull request by tongyu-microsoft**
This PR removes all .bind() in rai repository. And set rules to block adding new .bind().

## Description
The UI refactor work is to remove all .bind() functions. And this PR is part 4.
part 1 PR link: https://github.com/microsoft/responsible-ai-toolbox/pull/1523.
part 2 PR link: https://github.com/microsoft/responsible-ai-toolbox/pull/1542.
part 3 PR link: https://github.com/microsoft/responsible-ai-toolbox/pull/1555.
The reason that we want to remove .bind() is that, a different callback is created each time the component renders. So we want to remove .bind() and set lint rule in the end to ban adding .bind() in the future.

## Checklist

- [ ] I have added screenshots above for all UI changes.
- [ ] I have added e2e tests for all UI changes.
- [ ] Documentation was updated if it was needed.

### CODE REVIEW

File: apps/dashboard/src/fairness/App.tsx [hunk 1]
```diff
@@ -36,4 +41,43 @@ export class App extends React.Component<IAppProps> {
...
+    request: IMetricRequest,
+    abortSignal?: AbortSignal
+  ): Promise<IMetricResponse> => {
+    const binSize = _.max(request.binVector) || 0;
+    const bins: number[] = new Array(binSize + 1)
+      .fill(0)
+      .map(() => Math.random() / 3 + 0.33);
+    const binBounds: IBounds[] = bins.map((bin) => {
+      return {
+        lower: bin - Math.pow(Math.random() / 3, 2),
```
Comment (imatiach-msft) [inline apps/dashboard/src/fairness/App.tsx:55]: nit: maybe move this repeating lower/upper computation to some small function?

File: libs/interpret-text/src/lib/TextExplanationDashboard/TextExplanationDashboard.tsx [hunk 2]
```diff
@@ -66,7 +66,6 @@ export class TextExplanationDashboard extends React.PureComponent<
         Utils.countNonzeros(this.props.dataSummary.localExplanations) / 2
       )
     };
-    this.changeRadioButton = this.changeRadioButton.bind(this);
```
Comment (imatiach-msft) [inline libs/interpret-text/src/lib/TextExplanationDashboard/TextExplanationDashboard.tsx:69]: thanks for removing this!

**Review by imatiach-msft — COMMENTED** [review]
Awesome work!  Great refactoring/fixes.

**Review by imatiach-msft — APPROVED** [review]

File: libs/interpret-text/src/lib/TextExplanationDashboard/TextExplanationDashboard.tsx (hunk 2 above)
Comment (tongyu-microsoft) [reply libs/interpret-text/src/lib/TextExplanationDashboard/TextExplanationDashboard.tsx:69]: Thanks! :)

File: apps/dashboard/src/fairness/App.tsx (hunk 1 above)
Comment (tongyu-microsoft) [reply apps/dashboard/src/fairness/App.tsx:55]: Thanks for the comment! Moved to small function!

**Review by RubyZ10 — APPROVED** [review]

**Pull request merged**